### PR TITLE
CHK-4029: Add onboarding banner.

### PR DESCRIPTION
**Pull request by boldcaitlin**
When Payment Booster is not enabled:
image

When Payment Booster is enabled:
image

### CODE REVIEW

File: view/adminhtml/web/css/onboard-styles.css
Comment (JosephLeedy) [inline view/adminhtml/web/css/onboard-styles.css:1]: This should be a LESS stylesheet rather than plain CSS.

**Review by JosephLeedy — CHANGES_REQUESTED** [review]

File: view/adminhtml/layout/adminhtml_system_config_edit.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-2columns-left"
+      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <css src="Bold_CheckoutPaymentBooster::css/onboard-styles.css"/>
```
[the remark this answers is not in the document]
Comment (boldcaitlin) [reply view/adminhtml/layout/adminhtml_system_config_edit.xml:5]: I time boxed this to just over half a day and don’t have the svg rendering as expected. 

As neither of us could find sufficient Adobe Commerce docs, this follows patterns established in other [bold modules](https://github.com/bold-commerce/adobe-commerce-bold-checkout/tree/main/view/adminhtml/web/css) & [adobe documentation](https://developer.adobe.com/commerce/frontend-core/guide/css/themes/) I’d like to leave this styling as is - any objections?

[the remark this answers is not in the document]
Comment (JosephLeedy) [reply view/adminhtml/layout/adminhtml_system_config_edit.xml:5]: IMHO, our modules are doing it wrong. Let me play with this for an hour or two and see if I can get it to work the right way.

[the remark this answers is not in the document]
Comment (JosephLeedy) [reply view/adminhtml/layout/adminhtml_system_config_edit.xml]: I had inline the SVG files with the style rules due to an apparent bug in Magento's LESS parser.

**Review by TimeBones — APPROVED** [review]

File: view/adminhtml/layout/adminhtml_system_config_edit.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (JosephLeedy) [reply view/adminhtml/layout/adminhtml_system_config_edit.xml]: That's not necessary. Static content only needs to be deploying in production mode, and doing so if needed is a normal part of the installation process.

**Review by JosephLeedy — APPROVED** [review]

File: view/adminhtml/layout/adminhtml_system_config_edit.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (boldcaitlin) [reply view/adminhtml/layout/adminhtml_system_config_edit.xml]: Even better! Thanks for confirming!

**Comment by JosephLeedy** [comment]
For future reference, the solution for the issue with the images is that `onboard-styles.less` should have been prefixed with an underscore (i.e. `_onboard-styles.less`), the import in `_module.less` needs an extension, and the `url` rule should look like this: `url('@{baseDir}Bold_CheckoutPaymentBooster/images/file.svg')`.

**Pull request merged**